pub/assets/js/hound.js

```diff
@@ -282,17 +282,7 @@ var Model = {
   },
 
   UrlToRepo: function(repo, path, line) {
-    var info = this.repos[repo],
-        url = info.url.replace(/\.git$/, ''),
-        anc = line ? '#L' + line : '';
-
-    // Hacky solution to fix _some more_ of the 404's when using SSH style URLs
-    var sshParts = /git@(.*):(.*)/i.exec(url);
-    if (sshParts) {
-      url = '//' + sshParts[1] + '/' + sshParts[2];
-    }
-
-    return url + '/blob/master/' + path + anc;
+    return UrlToRepo(this.repos[repo], path, line);
```

Comment:
This is a little confusing, since it's calling a function in the global scope with the same name as this one.